bazel/version/BUILD

```diff
@@ -25,21 +48,38 @@ string_flag(
     ],
 )
 
+# When `--pre_release=rc` is used, `--rc_number=N` set's the release candidate
+# number to `N`.
+#
+# An error if used without `--pre_release=rc`.
```

Comment:
Optional, just suggesting an alternative boilerplate that's a little more terse. (not going to suggest this everywhere)

```suggestion
# `--rc_number=N` sets the release candidate number to `N`. Requires `--pre_release=rc`.
```